admin/load.php

```diff
@@ -266,10 +280,11 @@ function( $a, $b ) {
  * @since 1.0.0
  *
  * @param string $module_file Absolute path to the main module file.
+ * @param string $module_dir  The module directory ({focus}/{module-slug}).
  * @return array|bool Associative array of parsed module data, or false on failure. Fields for every module include
  *                    'name', 'description', 'focus', and 'experimental'.
  */
-function perflab_get_module_data( $module_file ) {
+function perflab_get_module_data( $module_file, $module_dir ) {
```

Comment:
I think passing `$module_dir` here is unnecessary, we could also get that inside this function from the `$module_file` via using a cobination of `dirname()` and `basename()`. That makes using the function simpler.

Reply:
👍🏼  Adjusted in [`f126464` (#58)](https://github.com/WordPress/performance/pull/58/commits/f1264644d7de67eb010224b0e10b49ad508adbb1)